src/main/java/com/minecolonies/coremod/client/gui/WindowHutMiner.java

```diff
@@ -21,16 +21,14 @@
 {
     private static final String LIST_LEVELS               = "levels";
     private static final String PAGE_LEVELS               = "levelActions";
-    private static final String BUTTON_PREVPAGE           = "prevPage";
-    private static final String BUTTON_NEXTPAGE           = "nextPage";
     private static final String BUTTON_CURRENTLEVEL       = "changeToLevel";
     private static final String VIEW_PAGES                = "pages";
     private static final String HUT_MINER_RESOURCE_SUFFIX = ":gui/windowHutMiner.xml";
-    private Button             buttonPrevPage;
-    private Button             buttonNextPage;
-    private int[]              levels;
-    private ScrollingList      levelList;
-    private BuildingMiner.View miner;
+    private       Button             buttonPrevPage;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "buttonPrevPage" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)